### PR TITLE
Added documentation to the main README

### PR DESCRIPTION
**Pull request by osher-sade**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,102 @@
...
+-   **typeDefs** (_any_) - The GraphQL schema written in SDL (Schema Definition Language).
+    This will be used in order to create your GraphQL API.
+-   **resolvers** (_any_) - The GraphQL resolvers that will be tied to your GraphQL schema.
+    This object contains functions and logic for the GraphQL engine to invoke when using fields from the schema.
+-   **port** (_number_) - Specify a port the `PolarisServer` should start the server on.
+-   **applicationProperties** (_ApplicationProperties - optional_) - Properties that describe your repository.
+    If you don't provide those properties, the core will put 'v1' in the version.
+-   **customMiddlewares** (_any[] - optional_) - Custom middlewares that can be provided the `PolarisServer` with.
+-   **customContext** (_(context: any, connection?: Connection) => any - optional_) - You can provide the `PolarisServer` your own custom context.
+    If you do not set your custom middleware, the core will use a default core.
```
Comment (chenshoo) [inline README.md]: didn't get what you wanted to say

File: README.md [hunk 2]
```diff
@@ -1 +1,102 @@
-# polaris-core
\ No newline at end of file
+![Polaris-logo](static/img/polaris-logo.png)
+
+# polaris-core
+
+[![NPM version](https://img.shields.io/npm/v/@enigmatis/polaris-core.svg?style=flat-square)](https://www.npmjs.com/package/@enigmatis/polaris-core)
+[![Build Status](https://travis-ci.com/Enigmatis/polaris-core.svg?branch=master)](https://travis-ci.com/Enigmatis/polaris-core)
+
+This is the core library of polaris. Through this core project we supply you with all of the polaris products.
```
Comment (itaykl) [inline README.md]: remove with

Comment (itaykl) [inline README.md]: add Overview bullet

**Pull request merged**